evm/src/executor/abi.rs

```diff
@@ -21,6 +21,10 @@ ethers::contract::abigen!(
             store(address,bytes32,bytes32)
             load(address,bytes32)(bytes32)
             ffi(string[])(bytes)
+            env_string(string)(string)
+            env_uint(string)(uint256)
+            env_int(string)(int256)
+            env_bool(string)(bool)
```

Comment:
These should be camel case as all the other cheatcodes

```suggestion
            env_string(string)(string)
            env_uint(string)(uint256)
            env_int(string)(int256)
            env_bool(string)(bool)
```

Reply:
thanks for catching this, fixed